influxdb/client_test.go

```diff
@@ -243,11 +253,12 @@ func TestClient_WriteLarge(t *testing.T) {
 	if err != nil {
 		t.Errorf("unexpected error.  expected %v, actual %v", nil, err)
 	}
-
+	wg.Add(1)
 	err = c.Write(bp)
```

Comment:
Is `Write` async here, and that's why we need the wait group?

Reply:
It isn't, but there was a bug in v1.5.8 that made it async (and fail for large payloads), this was added as a regression test around that bug.